lmdeploy/pytorch/messages.py

```diff
@@ -72,7 +72,10 @@ def from_gen_config(self, gen_config: EngineGenerationConfig):
             logger.warning('`top_p` has to be a float > 0 and < 1'
                            f' but is {top_p}')
             top_p = 1.0
-        if temperature <= 0:
+        if temperature == 0:
```

Comment:
@grimoire what's the minimum temperature in torch engine?

Reply:
https://github.com/InternLM/lmdeploy/blob/a06174f836882d853d4eb18519c2245c2a7eae8c/lmdeploy/pytorch/messages.py#L75-L78

zero or negative temperature would be set to 1

Reply:
Only accept positive temperature.

Reply:
> Only accept positive temperature.

Users may use temperature 0 in the sampling parameter, and currently this modification meets this requirement.

Reply:
Accept

Reply:
> Accept

cc @lvhan028